controllers/managedocs_controller.go

```diff
@@ -39,25 +40,25 @@ import (
 )
 
 const (
-	storageClusterName         = "ocs-storagecluster"
-	addonSecretName     string = "addon-ocs-converged-parameters"
-	storageClassSizeKey string = "size"
+	storageClusterName   = "ocs-storagecluster"
+	addonParamSecretName = "addon-ocs-converged-parameters"
+	storageClassSizeKey  = "size"
 )
 
 // ManagedOCSReconciler reconciles a ManagedOCS object
 type ManagedOCSReconciler struct {
 	client.Client
-	Log    logr.Logger
-	Scheme *runtime.Scheme
-	ctx    context.Context
-
+	Log              logr.Logger
+	Scheme           *runtime.Scheme
+	ctx              context.Context
 	managedOCS       *v1.ManagedOCS
 	ConfigSecretName string
 }
 
 // +kubebuilder:rbac:groups=ocs.openshift.io,resources=managedocs,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=ocs.openshift.io,resources=managedocs/status,verbs=get;update;patch
-// +kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=ocs.openshift.io,resources=storageclusters,verbs=*
```

Comment:
Same as my last RBAC related comment

Reply:
That will be taken care of when it is rebased.